### PR TITLE
feat: add icon display

### PR DESCRIPTION
**Pull request by Kun8018**
feat: add icon display

### CODE REVIEW

File: app/assets/components/Expand/ExpandForm/index.tsx [hunk 1]
```diff
@@ -138,47 +143,50 @@ class Expand extends React.Component<IProps, IState> {
...
-            trackEvent('explore', 'expand', 'ajax fail');
-            if (e.message) {
-              message.error(e.message);
-            } else {
-              message.info(intl.get('common.noData'));
-            }
-          },
-        );
+      if (err) {
+        console.error(err);
```
Comment (hetao92) [inline app/assets/components/Expand/ExpandForm/index.tsx]: remove console

File: app/assets/components/Expand/ExpandForm/CustomSet.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              customIcon={customIcon}
+              customColor={customColor}
+              editing={true}
+            />
+          }
+          trackCategory="explore"
+          trackAction="color_picker"
+          trackLabel="from_panel"
+        />
+      </>
```
Comment (jerryliang64) [inline app/assets/components/Expand/ExpandForm/CustomSet.tsx]: code format indent

File: app/assets/config/explore.ts [hunk 3]
```diff
@@ -222,6 +222,6 @@ export const DEFAULT_EXPLORE_RULES = {
   edgeDirection: 'outgoing',
   stepsType: 'single',
   step: 1,
-  vertexColor: 'groupByTag',
+  vertexSets: 'groupByTag',
```
Comment (jerryliang64) [inline app/assets/config/explore.ts]: vertexSets may be not a good name, I don't know what's the meaning of it

Comment (jerryliang64) [reply app/assets/config/explore.ts]: how about vertexStyle? In addition, `groupByTag` can be more detailed to `colorGroupByTag`

File: app/assets/components/Expand/ExpandForm/CustomSet.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
Comment (jerryliang64) [inline app/assets/components/Expand/ExpandForm/CustomSet.tsx]: how about `CustomSet` to `CustomStyle`?

**Comment by jerryliang64** [comment]
The suffix of `Set` maybe have other meaning which confuses readers, maybe `Style` is a better choice

**Review by jerryliang64 — APPROVED** [review]
LGTM

**Pull request merged**